### PR TITLE
update ssb-bfe

### PR DESCRIPTION
**Pull request by mixmix**
context - ssb-bfe is being updated, and there are some breaking changes in names

this PR:
- fixes the name changes
- refactored util function:
    - readability

## TODO
- install `ssb-bfe` when it's published, then merge this

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "envelope-spec": "^1.0.0",
     "futoin-hkdf": "^1.3.2",
     "sodium-native": "^3.2.0",
-    "ssb-bfe": "^1.0.0"
+    "ssb-bfe": "github:ssb-ngi-pointer/ssb-bfe#spec_dep"
```
Comment (mixmix) [inline package.json:21]: TODO - install real version

**Review by arj03 — APPROVED** [review]
Looks good. Nice to the helper function in action :)

**Comment by mixmix** [comment]
Thanks. Yeah if you're up for meeting, updating modules and publishing this one too that would be awesome

**Comment by staltz** [comment]
> Yeah if you're up for merging, updating modules and publishing this one too that would be awesome

I could do this, but I haven't updated package-lock.json, and I also haven't touched (been responsible for) envelope-js before, so I'll let you do it.

**Comment by arj03** [comment]
I'll fix the last stuff and publish

**Comment by mixmix** [comment]
Thanks!

**Pull request merged**